apps/web/components/SubpageMainContent/SubpageMainContent.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, ReactNode } from 'react'
+import {
+  Box,
+  GridContainer,
+  GridRow,
+  GridColumn,
+} from '@island.is/island-ui/core'
+
+interface SubpageMainProps {
+  main: ReactNode
+  image?: ReactNode
+}
+
+export const SubpageMainContent: FC<SubpageMainProps> = ({ main, image }) => {
+  return (
+    <Box>
+      <GridContainer>
+        {!image && (
+          <GridRow>
+            <GridColumn span="12/12">{main}</GridColumn>
+          </GridRow>
+        )}
+        {image && (
```

Comment:
Could simplify this a tiny bit using a ternary 
`{ image ? <some image content> : <no image content> }`